src/test/java/uk/gov/hmcts/darts/cases/mapper/HearingEntityToCaseHearingTest.java

```diff
@@ -29,12 +42,18 @@ void beforeAll() {
         objectMapper = objectMapperConfig.objectMapper();
     }
 
+    @BeforeEach
+    void beforeEach() {
+        lenient().when(transcriptionDocumentRepository.findByTranscriptionIdAndHiddenTrueIncludeDeleted(anyInt()))
+            .thenReturn(Collections.emptyList());
```

Comment:
Should add some verify(transcriptionDocumentRepository).findByTranscriptionIdAndHiddenTrueIncludeDeleted(expectedInt) to ensure the correct values are being passed to this for tests which call this method.

For tests that do not call this we should verifyNoInteractions

Reply:
done